src/components/dashboard/menu-table/popup-forms.js

```diff
@@ -426,7 +429,19 @@ const NewCategoryForm = (props) => {
             <ModalBackground />
             <StyledModal>
                 <Container>
-                    <FormInput placeholder='category' name='category' onChange={(event) => { setName(event.target.value) }} />
+                    <DishFormTitle>Add Category</DishFormTitle>
+                    <DishFormSubtitle>Category Name</DishFormSubtitle>
+                    <DishFormInput placeholder='Type a category name (e.g. Appetizers, Entrees)...' name='category' value={ name } onChange={(event) => { setName(event.target.value) }}/>
+                    <Divider/>
+                    <DishFormSubtitle>Description</DishFormSubtitle>
+                    <DishFormTextArea
+                      placeholder="Change description..."
+                      value={ description }
```

Comment:
Shall we use a different prompt/title so that it's clear to the user that category descriptions are optional and they don't have to make things up. I think we might've discussed this during our meeting last week but I don't remember if we have a conclusion or not...

Reply:
added optional for the subtitle!